scripts/gen-s-parser.py

```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/env python3.6
```

Comment:
Do you need python3 here?

Reply:
Not really, but I like its f-strings and this script will never need to be run by end users.

Reply:
Can this be `python3` (without `.6`)?